public/src/components/cols.js

```diff
@@ -39,12 +39,12 @@ export default React.createClass({
     let sum = 0
     let cols = []
     for (let key in zone) {
-      let items = zone[key].map(card =>
-        d.div({
-          className: card.color,
+      let items = zone[key].map(card => {
+        return d.div({
```

Comment:
you don't have to explicitly return from functions with a single expression:

https://github.com/lukehoban/es6features#arrows